src/router.js

```diff
@@ -7,6 +7,7 @@ import Home from './views/Home'
 import Maps from './views/Maps'
 import Viewer from './views/Viewer'
 import Combine from './views/Combine'
+import Toegankelijkheid from './views/Toegankelijkheid'
```

Comment:
Component names in English for consistency

Reply:
The component name now reflects the url, do you want:
a) Keep /toegankelijkheid and have the component be Accessibility
b) Rename the url to /accessibility so it match the component name
c) Leave it as is